web/src/definitions.ts

```diff
@@ -1,12 +1,12 @@
 const Definitions = {
-  studyName: 'Provide a study name to associate with your samples.',
+  studyName: 'The Study Information page requires a valid ‘Study Name’. We recommend the use of standardized, informative study names as described by the GOLD team (Mukherjee et. al., 2023).',
```

Comment:
I'm not sure that the user needs to be reminded that they're on the Study Information page.

```suggestion
  studyName: 'A Study Name is required. We recommend the use of standardized, informative study names as described by the GOLD team (Mukherjee et. al., 2023).',
```